artist-traits/src/main/kotlin/com/uber/artist/traits/SuppressNullabilityInitializerTrait.kt

```diff
@@ -0,0 +1,23 @@
+package com.uber.artist.traits
+
+import com.google.auto.service.AutoService
+import com.squareup.javapoet.AnnotationSpec
+import com.squareup.javapoet.ClassName
+import com.squareup.javapoet.MethodSpec
+import com.squareup.javapoet.TypeSpec.Builder
+import com.uber.artist.api.Trait
+
+@AutoService(Trait::class)
+class SuppressNullabilityInitializerTrait: Trait {
```

Comment:
nit: space after the class name ([see point 5](https://android.github.io/kotlin-guides/style.html#horizontal))